### PR TITLE
Modify rule S2970: Improve description of supported test library functions

### PR DESCRIPTION
**Pull request by martin-strecker-sonarsource**
Implementation: https://github.com/SonarSource/sonar-dotnet/issues/6662

[S2970](https://sonarsource.github.io/rspec/#/rspec/S2970/)

### CODE REVIEW

**Review by gregory-paidis-sonarsource — APPROVED** [review]
LGTM!

**Review by cristian-ambrosini-sonarsource — APPROVED** [review]
LGTM

File: rules/S2970/csharp/rule.adoc [hunk 1]
```diff
@@ -1,19 +1,40 @@
-include::../description.adoc[]
+It is very easy to write incomplete assertions when using some test frameworks. This rule enforces complete assertions in the following cases:
+
+* MSTest: https://learn.microsoft.com/dotnet/api/microsoft.visualstudio.testtools.unittesting.assert.that[`Assert.That`] is not followed by an assertion invocation (also https://learn.microsoft.com/dotnet/api/microsoft.visualstudio.testtools.unittesting.stringassert.that[`StringAssert.That`] and https://learn.microsoft.com/dotnet/api/microsoft.visualstudio.testtools.unittesting.collectionassert.that[`CollectionAssert.That`]).
+* Fluent Assertions: https://fluentassertions.com/introduction[`Should()`] is not followed by an assertion invocation.
+* NFluent: https://www.n-fluent.net[`Check.That()`] is not followed by an assertion invocation
```
Comment (cristian-ambrosini-sonarsource) [inline rules/S2970/csharp/rule.adoc:5]: Nitpick comment: both this sentence and the next one lack the full stop at the end.

**Review by cristian-ambrosini-sonarsource — APPROVED** [review]
LGTM! Left one nitpick comment

**Pull request merged**